js/components/FileModal.tsx

```diff
@@ -11,6 +11,8 @@ import { FileChooser, FilePickerAcceptType, FileSystemFileHandleLike } from './F
 import { noAwait } from './util/promises';
 import { useHash } from './hooks/useHash';
 
+import styles from './css/FileModal.module.css';
```

Comment:
I like setting the style import off from the rest of the imports. Should we do that everywhere or do you think it doesn't matter?